.gitignore

```diff
@@ -26,3 +26,6 @@ coverage.txt
 
 # patron binaries
 /cmd/patron/patron
+
```

Comment:
This new entry on the `.gitignore` is to protect against committing temporary files created by the new script `examples/services.sh`.